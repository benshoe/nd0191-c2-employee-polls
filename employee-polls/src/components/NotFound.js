import PropTypes from "prop-types";

const NotFound = ({props}) => {
    if(props) {
        return <h1>Question with id: {props} not found</h1>
    }
    return <h1>404: Page not found</h1>
}

export default NotFound;

NotFound.propTypes ={
    props: PropTypes.object,
}
