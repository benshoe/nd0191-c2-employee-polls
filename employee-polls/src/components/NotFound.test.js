import renderer from "react-test-renderer";
import NotFound from "./NotFound";

describe('Test NotFound', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(<NotFound props={"1234"} />)
            .toJSON();

            expect(tree).toMatchSnapshot();
    });
});
