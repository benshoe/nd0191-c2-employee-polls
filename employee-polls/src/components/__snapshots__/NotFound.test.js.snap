// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test NotFound renders correctly 1`] = `
<h1>
  Question with id: 
  1234
   not found
</h1>
`;
